src/libraries/System.Runtime.Numerics/src/System/Numerics/BigNumber.cs

```diff
@@ -511,6 +515,96 @@ private static ParsingStatus HexNumberToBigInteger(ref BigNumberBuffer number, o
             }
         }
 
+        private static ParsingStatus BinNumberToBigInteger(ref BigNumberBuffer number, out BigInteger result)
+        {
+            if (number.digits is null || number.digits.Length < 2)
+            {
+                result = default;
+                return ParsingStatus.Failed;
+            }
+
+            const int DigitsPerBlock = 32;
```

Comment:
this should be seeded off of `kcbitUint` or otherwise asserted to be the same

Reply:
`kcbitUint` is private field of `BigInteger` which is not accessible from current BigNumber, Please correct my understand for this comment, thanks.

Reply:
We can make it `internal` so that it can be shared here and avoid having conflicting information in the future if it ever changes

Reply:
Got it, fixed.